OpenTESArena/CMakeLists.txt

```diff
@@ -78,6 +78,10 @@ IF (WIN32)
     ADD_DEFINITIONS("-D_SCL_SECURE_NO_WARNINGS=1")
 ENDIF()
 
+# Copy over required files
+FILE(COPY ${CMAKE_SOURCE_DIR}/data DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
Gotcha. I just found out that `${CMAKE_CURRENT_SOURCE_DIR}` points to the subdirectory, so definitely not what we want. New commit is submitted.